src/electionguard-ui/ElectionGuard.UI.Lib/Models/ElectionPartialKeyBackup.cs

```diff
@@ -7,8 +7,16 @@ public class ElectionPartialKeyBackup : DisposableBase
 {
     // TODO: implement IElectionGuardian
 
+    /// <summary>
+    /// The guardian id of the owner who created this backup
+    /// </summary>
     public string OwnerId { get; set; }
 
+    public ulong OwnerSequenceOrder { get; set; }
```

Comment:
added this field to make it easier to access from the backups directly

Reply:
We will have to make sure that people know that they need to clean out their db... or that the old elections will not work.